packages/client/src/app/group/services/group.service.js

```diff
@@ -15,4 +15,20 @@ export class GroupService {
   async getAllForUser(userId) {
     return await HttpRequestService.instance().getRequest('/api/user/' + userId + '/group');
   }
+
+  /** @return {Promise<Group>} */
+  async createForUser(userId, name) {
+    return await HttpRequestService.instance().postRequest('/api/group/', {
+      name: name,
+      user_id: userId
```

Comment:
I think that over most of the request parameters we use camelCase format instead of snake_case